diff --git a/base/BUILD.gn b/base/BUILD.gn
index bd94a2c1a236086fdc99f0a2ff1dc8422f14579b..903844ec5b5891e5a5a8674dc95f984f1a8379a1 100644
--- a/base/BUILD.gn
+++ b/base/BUILD.gn
@@ -4333,6 +4333,7 @@ if (is_android) {
       "android/java/src/org/chromium/base/task/TaskTraits.java",
       "android/java/src/org/chromium/base/task/TaskTraitsExtensionDescriptor.java",
     ]
+    sources += brave_java_base_sources
 
     if (!is_cronet_build) {
       sources += [ "android/java/src/org/chromium/base/IntentUtils.java" ]
