diff --git a/components/bookmarks/browser/bookmark_model.h b/components/bookmarks/browser/bookmark_model.h
index d811a6527629bae4ada9a2a76a254156a8c309f7..556b28b5b7d38d23eadf4f3cb6ca08684526483f 100644
--- a/components/bookmarks/browser/bookmark_model.h
+++ b/components/bookmarks/browser/bookmark_model.h
@@ -350,6 +350,7 @@ class BookmarkModel : public BookmarkUndoProvider,
     return weak_factory_.GetWeakPtr();
   }
 
+  BRAVE_BOOKMARK_MODEL_H
  private:
   friend class BookmarkCodecTest;
   friend class BookmarkModelFaviconTest;
