diff --git a/components/content_settings/core/common/content_settings_types.h b/components/content_settings/core/common/content_settings_types.h
index eb7eb470c46eefbb93a2929bbc46ec81f3813854..510bbaf211f8684732673aba84e55e384afe4ca5 100644
--- a/components/content_settings/core/common/content_settings_types.h
+++ b/components/content_settings/core/common/content_settings_types.h
@@ -311,6 +311,21 @@ enum class ContentSettingsType : int32_t {
   // being specific to a frame.
   TOP_LEVEL_STORAGE_ACCESS,
 
+  BRAVE_START,
+  BRAVE_ADS = BRAVE_START,
+  BRAVE_COSMETIC_FILTERING,
+  BRAVE_TRACKERS,
+  BRAVE_HTTP_UPGRADABLE_RESOURCES,
+  BRAVE_FINGERPRINTING_V2,
+  BRAVE_SHIELDS,
+  BRAVE_REFERRERS,
+  BRAVE_COOKIES,
+  BRAVE_SPEEDREADER,
+  BRAVE_ETHEREUM,
+  BRAVE_SOLANA,
+  BRAVE_GOOGLE_SIGN_IN,
+  BRAVE_HTTPS_UPGRADE,
+  BRAVE_REMEMBER_1P_STORAGE,
   NUM_TYPES,
 };
 
