diff --git a/ios/chrome/app/BUILD.gn b/ios/chrome/app/BUILD.gn
index 100977b15e6fd30b1a26d1f883d71e115c458d4d..18eba76ca289d1d2e4c7ccdc739000dec431b0b8 100644
--- a/ios/chrome/app/BUILD.gn
+++ b/ios/chrome/app/BUILD.gn
@@ -737,8 +737,7 @@ ios_app_bundle("chrome") {
   # (see https://crbug.com/1394553 for details of the build failure). This
   # is a stop-gap solution.
   if (target_environment == "catalyst" ||
-      ios_provider_target !=
-      "//ios/chrome/browser/providers:chromium_providers") {
+      !ios_chrome_links_with_lottie_framework) {
     assert_no_deps += [
       "//ios/third_party/lottie:lottie+bundle",
       "//ios/third_party/lottie:lottie+link",
